query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/NestedDisconnectMutationInsideUpsertSpec.scala

```diff
@@ -818,7 +821,8 @@ class NestedDisconnectMutationInsideUpsertSpec extends FlatSpec with Matchers wi
          |}
       """,
       project,
-      errorCode = 3041
+      errorCode = 0, // 3041
+      errorContains = """Error occurred during query execution:\nInterpretationError(\"Error for binding \\'5\\': RecordsNotConnected { relation_name: \\\"UserFollows\\\", parent_name: \\\"User\\\", child_name: \\\"User\\\"""
```

Comment:
Not sure these (in this file) should be user facing with an error code

Reply:
I think they should